rust/src/main.rs

```diff
@@ -1,135 +1,230 @@
-use futures_lite::future::FutureExt;
-use std::env::{current_dir, vars};
-use std::sync::Arc;
-use tokio::signal;
+//! Echo bot example in Rust.
+//!
+//! This is a bot example which echo's back the text of any message it receives in direct
```

Comment:
"which echoes" back? maybe it is my bad English but I don't understand "which echo's back"